### ISSUE TITLE
Product's Picture overview doesn't work when adding Product's Gallary

### ISSUE THREAD

**Issue by Seeder101** [woocommerce/woocommerce#20284]
**Describe the bug**
 After upgrading to 3.4 Product's picture overview doesn't work when adding pictures to the Gallary of the product 
a blank white page you will see(or really it depends on the background theme you're using) if you want to see the picture you have to click on the small magnifying glass with +  

**To Reproduce**
1. Go to 'WooCommerce > Products > Select any Product you want'
2. Scroll down to 'Product Image on the down-right-side of the Page'
3. Click on 'Add Product Gallary Images'
4. Add a set of pictures and click on update and do a preview of the product 

**Screenshots**
sadly I don't have screenshots atm because I deleted the product's gallary images and things are working fine but it's annoying specially when you have different set of pictures of the same product with different angles 

**Expected behavior**
nothing will show up in the product's picture overview and it will be working if you click on the small magnifying glass with + 

**Isolating the problem (mark completed items with an [x]):**
- [ ] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [x] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/storefront/).
- [x] I can reproduce this bug consistently using the steps above.

**WordPress Environment**
<details>
```
Copy and paste the system status report from **WooCommerce > System Status** in WordPress admin.
```
</details>

**Comment by mikejolley** [comment]
Without a status report or link to a broken product there isn't much I can do here; on our test sites this problem doesn't exist. It sounds like it could be one of two things;

1. Missing thumbnail image (tried regenerating thumbnails?)
2. Theme's gallery display issue

My working test:

![slack - a8c 2018-05-29 11-34-30](https://user-images.githubusercontent.com/90977/40653836-48cbcc2a-6334-11e8-97d3-40a1efc620d8.png)

If you want to give access to someone so they can take a proper look, please contact the helpdesk. https://woocommerce.com/contact-us/

Thanks

**Comment by Seeder101** [comment]
I've updated the issue with a screenshot and I filled out the wordpress environment status report from woocommerce also here is a direct link to a sample product [https://banafa.com.sa/product/%d8%a7%d8%b3%d9%81%d9%86%d8%ac-%d8%aa%d9%86%d8%b8%d9%8a%d9%81-%d9%83%d9%88%d8%b1%d9%8a/](url)

**Comment by mikejolley** [comment]
@Seeder101 Can you try disabling all plugins except for WC and Super cache, and flush super cache?

Then repeat with twenty twelve theme. 

Only on this page I see a 400% width being applied in the gallery to the woocommerce-product-gallery__wrapper figure element. 

https://banafa.com.sa/product/%D8%A7%D8%B3%D9%81%D9%86%D8%AC-%D8%AA%D9%86%D8%B8%D9%8A%D9%81-%D9%83%D9%88%D8%B1%D9%8A/

Other pages don't have this.

**Comment by Seeder101** [comment]
@mikejolley so I've tried all the troubleshooting above and none of that worked. ironically when I switch my entire WP to English (LTR) the gallary pictures show up perfectly fine but when I switch back to Arabic (RTL) it doesn't work... so it might this thing have to do with RTL ? 

Thank you

**Comment by mikejolley** [comment]
https://github.com/woocommerce/woocommerce/pull/20321

**Comment by Seeder101** [comment]
I can confirm the issue is being fixed after upgrading to 3.4.2. 
Thank you again
